test/benchmarks/standalone/benchmark_text_input.cc

```diff
@@ -281,6 +281,22 @@ BENCHMARK_CAPTURE(benchmark_multi, ccb_adf_diff_char_no_interactions,
 BENCHMARK_CAPTURE(benchmark_multi, ccb_adf_diff_char_interactions, gen_ccb_examples(50, 7, 3, 6, 3, 4, 14, 2, false, 3),
     "--ccb_explore_adf --quiet -q ::")
     ->MinTime(15.0);
+// CCB predict with quadratic interactions
+// 20 slots, 20 context features,  30 actions, 20 action features each
+BENCHMARK_CAPTURE(benchmark_multi_predict, ccb_adf_diff_char_interactions_predict, gen_ccb_examples(
```

Comment:
How much does adding this benchmark increase our CI time by?

Reply:
we could probably remove some of the many LAS ones in this file to reduce the overall time, not all of them are needed and make some room for other benchmarks